go/keymanager/api/api.go

```diff
@@ -53,6 +53,19 @@ var (
 	initResponseContext = signature.NewContext("oasis-core/keymanager: init response")
 )
 
+const (
+	// GasOpUpdateKeyManager is the gas operation identifier for key manager
+	// policy updates costs.
+	GasOpUpdateKeyManager transaction.Op = "update_keymanager"
```

Comment:
This should probably be called `GasOpUpdatePolicy`.

Reply:
Should I also rename `"update_keymanager"` to `"update_policy"` and then remove `"update_keymanager"` in the fix genesis?

Reply:
Yeah I think that makes the most sense.